frontend/src/locales/scripts/en.json5

```
@@ -595,9 +597,9 @@
     alt: 'Cover art for "{title}" by {creator}',
   },
   "audio-track": {
-    "aria-label": "{title} - Audio Player",
-    "aria-label-interactive": "{title} - audio player - press the spacebar to play and pause a preview of the audio",
-    "aria-label-interactive-seekable": "{title} - audio player - press the spacebar to play and pause a preview of the audio; use the left and right arrow keys to seek through the track.",
+    "aria-label": "Audio: {title}",
+    "aria-label-interactive": "Audio: {title} - interactive player - press the spacebar to play and pause a preview of the audio",
```

```suggestion
    "aria-label-interactive": "Audio: {title} - interactive player - press the space bar to play and pause a preview of the audio",
```
On Voiceover on Mac, the no space variant is read as "spuh-seh-bar".

---

@obulat this might depend on the default system voice—"spacebar" was read correctly for me.